src/nimony/sem.nim

```diff
@@ -861,9 +867,11 @@ proc considerTypeboundOps(c: var SemContext; m: var seq[Match]; candidates: FnCa
     m.add createMatch(addr c)
     sigmatch(m[^1], candidate, args, genericArgs)
 
-proc requestRoutineInstance(c: var SemContext; origin: SymId; m: var Match;
+proc requestRoutineInstance(c: var SemContext; origin: SymId;
+                            typeArgs: var TokenBuf;
+                            inferred: var Table[SymId, Cursor];
```

Comment:
I doubt these are used as `var`

Reply:
`inferred` is moved. I have removed var from `typeArgs`